ubcpi/static/css/ubcpi.css

```diff
@@ -349,17 +479,66 @@ textarea.pi-options {
 }
 
 .correct-answer-option {
-    margin-bottom: 20px;
+    margin-top: 20px;
+    margin-bottom: 5px;
+}
+
+.show-correct {
```

Comment:
I'm worried the name of this selector is too generic might clobber other names? (same with `.show-incorrect` )